rdrf/rdrf/views/patients_listing.py

```diff
@@ -89,6 +91,7 @@ def get_columns(self):
             ColumnDiagnosisProgress(_("Diagnosis Entry Progress"), "patients.can_see_diagnosis_progress"),
             ColumnDiagnosisCurrency(_("Updated < 365 days"), "patients.can_see_diagnosis_currency"),
             ColumnGeneticDataMap(_("Genetic Data"), "patients.can_see_genetic_data_map"),
+            ColumnPatientStage(_("Stage"), "patients.can_see_data_modules"),
```

Comment:
Can we add this only if there is at least one registry with stages feature on?

Reply:
It's possible. Need to look into it.